### PR TITLE
Proposal, sections 2 and 3

### PR DESCRIPTION
**Pull request by techrah**
- Section 2: Description of the Data
- Section 3: Research Questions and Usage Scenarios

### CODE REVIEW

**Comment by fkhan72** [comment]
I am pasting here my version of section 2 as I worked on it earlier. Ryan, feel free to combine the two versions 

> The dataset contains the record of around 3200 movies. For each movie, we have different types of information available, starting from basic information (`Title`, `Major_Genre`, `Release_Date`, `Running_Time_min`, `Director`) to some miscellaneous data (`Distributor`, `Creative_Type`, `Source`). The data also includes the cost and revenue (`US_Gross`, `Worldwide_Gross`, `Production_Budget`, `US_DVD_Sales`) along with the ratings (`MPAA_Rating`, `Rotten_Tomatoes_Rating`, `IMDB_Rating`, `IMDB_Votes`) for each movie. For the app, the movies will be searched based on `Major_Genre`, `MPAA_Rating` and `Release_Date`, and the results will be displayed based on `Worldwide_Gros Interactively, the viewer rating for each movie will be displayed based on a new variable called `overall_rating` using the information from `Rotten_Tomatoes_Rating` and `IMDB_Rating`.

**Comment by techrah** [comment]
OK, I will revise mine and add any missing information.

**Comment by techrah** [comment]
I've updated section 2. However, I am hesitant to add too many more column names based on this excerpt from the `milestone.md` document:

> Please note, if your dataset has a lot of columns and you plan to visualize them all, provide a high level descriptor of the variable types. For example, indicate that the dataset contains a variety of demographic variables and provide a brief list rather than stating and describing every single variable. You may also want to consider visualizing a smaller set of variables given the short duration of this project.

I interpret this to say:
- don't give an exhaustive list of column names
- not really necessary to mention column names that are not going to be visualized or otherwise used

I also did not mention how we are going to use them in the app. I think this section is only about describing the dataset itself.

Let me know if you still think we need further changes.

File: proposal.md [hunk 1]
```diff
@@ -6,11 +6,18 @@ TBA
...
+This "movies" dataset is from the [Vega Datasets][1] collection. There are about 3200 movies with release years between 1920 to 2016. For each movie, we have different types of information available, starting from basic information (`Title`, `Major_Genre`, `Release_Date`, `Running_Time_min`, `Director`) to some miscellaneous data (`Distributor`, `Creative_Type`, `Source`). Financial information includes US & worldwide gross income, US DVD sales and production budget. Movies are categorized by a single `Major_Genre` (e.g. Action, Comedy), a `Creative_Type` (e.g. Fantasy, Super Hero) and an `MPAA_Rating` which are [ratings by the Motion Picture Association of America][2] (e.g. G, PG, R). There are up to two ratings per movie in the variables `Rotten_Tomatoes_Rating` and `IMDB_Rating`.
```
Comment (fkhan72) [inline proposal.md:9]: Looks good to me. I think its good for merging

File: proposal.md [hunk 2]
```diff
@@ -6,11 +6,18 @@ TBA
...
 ## Research Questions and Usage Scenarios
 
-TBA
+- What are the top US grossing movies based on genre, MPAA rating and release year range?
+- How do these movies rate among other movies (in the dataset)?
+
+Bob is an avid movie watcher and would like to discover past movies that he may have missed or re-discover movies that he's enjoyed in the past. He wants an easy way to narrow down the vast selection of movies available so he can spend more time watching movies as opposed to searching for them.
+
+When Bob uses this "Seek-a-Movie" app, he'll be presented with two charts and some filter controls. The main chart will be interactive and will display the top movies by US gross income, filtered by all the user filter controls. The secondary chart, a scatter plot, displays Rotten Tomatoes ratings vs. IMBD ratings for the entire database. The ratings for the top movies listed in the first chart will be highlighted. This allows bob to visualize where the ratings for the filtered movie selection fall among all other ratings.
+
```
Comment (fkhan72) [inline proposal.md:19]: We need to clarify if selecting a movie will highlight the ratings or top movies ratings will be highlighted automatically

Comment (techrah) [reply proposal.md:19]: Ok, I've added the following sentence:

> If a particular movie is selected, the corresponding rating dot in the scatter plot will be emphasized and all remaining dots will be faded.

**Pull request merged**